plugin/audit_log/tests/mtr/audit_log_filter_users.test

```diff
@@ -71,6 +73,13 @@ SET GLOBAL audit_log_exclude_accounts= 'user1@localhost,, user22@127.0.0.1,admin
 
 SET GLOBAL audit_log_exclude_accounts= NULL;
 
+# testing case sensitivity
```

Comment:
Case sensitivity also needs testing for the username part too: Foo vs foo, filter just one of them